README.md

```diff
@@ -61,6 +65,8 @@ Manga Title.ext
 [Clone the repository](https://github.com/jvlflame/Fakku-Library/archive/master.zip) and extract the
 files to a directory of your choice.
 
+[Download the chromedriver](https://chromedriver.chromium.org/downloads) version that matches your version of chrome as well as the [Selenium WebDriver for C#](https://goo.gl/uJJ5Sc). Extract `chromedriver.exe` and `WebDriver.dll` (from inside `\selenium-dotnet-3.14.0.zip\dist\Selenium.WebDriver.3.14.0.nupkg\lib\net45\`) to a writable path (by default, it tries `C:\Selenium`).
```

Comment:
Maybe add extra note to rename the `.nupkg` as `.zip` so non-technical users don't get confused by the initial filetype extension.